relayer/relay.go

```diff
@@ -155,12 +155,15 @@ func (r *Relayer) StartBlockProcessors(ctx context.Context, errorChan chan error
 
 func (r *Relayer) StartRouter(ctx context.Context, flushInterval time.Duration) {
 	routeTimer := time.NewTicker(types.RouteDuration)
-	flushTimer := time.NewTicker(flushInterval)
-	heightTimer := time.NewTicker(HeightSaveInterval)
-	cleanMessageTimer := time.NewTicker(DeleteExpiredInterval)
+	flushTimer := time.NewTicker(1 * time.Second)
+	heightTimer := time.NewTicker(1 * time.Second)
```

Comment:
Do we want to immediately save height to latest height upon startup or how is it controlled ?

Reply:
yeah, this does not harm us. this helps us for recovery.

Reply:
So, what happens if we have blocks to sync and we get some issues during processing the backlog say after 3 seconds ?

Reply:
that's fine. We have protection from #329.

Reply:
either way this is best to remove.